sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql

```diff
@@ -0,0 +1,84 @@
+WITH
+  combined_urlbar_picked AS (
+  SELECT
+    submission_date,
+    client_id,
+    default_search_engine,
+    app_version,
+    normalized_channel,
+    locale,
+    [ STRUCT("autofill" AS type,
+      scalar_parent_urlbar_picked_autofill_sum AS index),
+    STRUCT("bookmark" AS type,
+      scalar_parent_urlbar_picked_bookmark_sum AS index),
+    STRUCT("dynamic" AS type,
+      scalar_parent_urlbar_picked_dynamic_sum AS index),
+    STRUCT("extension" AS type,
+      scalar_parent_urlbar_picked_extension_sum AS index),
+    STRUCT("formhistory" AS type,
+      scalar_parent_urlbar_picked_formhistory_sum AS index),
+    STRUCT("history" AS type,
+      scalar_parent_urlbar_picked_history_sum AS index),
+    STRUCT("keyword" AS type,
+      scalar_parent_urlbar_picked_keyword_sum AS index),
+    STRUCT("remotetab" AS type,
+      scalar_parent_urlbar_picked_remotetab_sum AS index),
+    STRUCT("searchengine" AS type,
+      scalar_parent_urlbar_picked_searchengine_sum AS index),
+    STRUCT("searchsuggestion" AS type,
+      scalar_parent_urlbar_picked_searchsuggestion_sum AS index),
+    STRUCT("switchtab" AS type,
+      scalar_parent_urlbar_picked_switchtab_sum AS index),
+    STRUCT("tabtosearch" AS type,
+      scalar_parent_urlbar_picked_tabtosearch_sum AS index),
+    STRUCT("tip" AS type,
+      scalar_parent_urlbar_picked_tip_sum AS index),
+    STRUCT("topsite" AS type,
+      scalar_parent_urlbar_picked_topsite_sum AS index),
+    STRUCT("unknown" AS type,
+      scalar_parent_urlbar_picked_unknown_sum AS index),
+    STRUCT("visiturl" AS type,
+      scalar_parent_urlbar_picked_visiturl_sum AS index) ] AS urlbar_picked
+  FROM
+   telemetry.clients_daily
+  WHERE
+    submission_date = @submission_date ),
+  count_picked AS (
+  SELECT
+    submission_date,
+    client_id,
+    COALESCE(SUM(index.value), 0) as count_picked_total,
+    mozfun.map.sum( ARRAY_AGG( STRUCT(type AS key,
+          index.value AS value) ) ) AS count_picked_by_type,
+    mozfun.map.sum( ARRAY_AGG(
+        -- TODO: decide whether convert to 1-based index for consistency
+        -- STRUCT(SAFE_CAST(index.key as INT64) + 1 AS key, index.value AS value)
+        STRUCT(SAFE_CAST(index.key AS INT64) AS key,
```

Comment:
Leave the -1's as -1. 1-Index everything else. We're moving to more data self service in the Data Org, and 1-indexing is easier for less technical users to understand.

```suggestion
        STRUCT(IF(SAFE_CAST(index.key AS INT64) < 0, SAFE_CAST(index.key AS INT64), SAFE_CAST(index.key AS INT64) + 1) AS key,
```